internal/pkg/core/backoff/equal_jitter.go

```diff
@@ -29,6 +30,7 @@ func NewEqualJitterBackoff(done <-chan struct{}, init, max time.Duration) Backof
 		done:     done,
 		init:     init,
 		max:      max,
+		nextRand: time.Duration(rand.Int63n(int64(init))),
```

Comment:
I think we can put a nolint directive for this line

Reply:
Will do, I went further and just turned off this linter in https://github.com/elastic/elastic-agent/pull/1478